OpenRA.Mods.Common/Traits/EjectOnDeath.cs

```diff
@@ -66,7 +66,11 @@ public void Killed(Actor self, AttackInfo e)
 			{
 				if (cp.Z > 0)
 				{
-					self.World.AddFrameEndTask(w => w.Add(new Parachute(pilot, cp)));
+					self.World.AddFrameEndTask(w =>
+					{
+						w.Add(pilot);
+						pilot.QueueActivity(new Activities.Parachute(pilot, cp));
```

Comment:
I think you can clean up the `using`s here, too, and then remove the `Activities`.

Bonus points if you add a `Desc()` that states the traits the `PilotActor` needs in the Info class.